src/Nexus.UI/Controls/UIContextMenu.razor

```diff
@@ -56,4 +42,12 @@
     {
         await IsOpenChanged.InvokeAsync(false);
     }
+
+    protected override void OnParametersSet()
```

Comment:
protected methods should be sorted before private methods:

https://stackoverflow.com/a/310967

Maybe that is something we can try to enforce in the editorconfig file